http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/AppRoutes.scala

```diff
@@ -247,7 +247,7 @@ object AppRoutes {
       "labels"
     )(x =>
       (x.cloudContext,
-       x.cloudContext,
+       x.cloudContext.asString,
```

Comment:
Will this return the property "cloudResource" from cloudContext?

We're expecting the property "cloudResource" here, at least for galaxy apps.

Reply:
@n8landolt yep...the `cloudContext` field will have what both `cloudResource` and `cloudProvider`